jsonschema/src/keywords/unevaluated_properties.rs

```diff
@@ -1094,6 +1329,11 @@ pub(crate) fn compile<'a>(
     }
 }
 
+fn boxed_errors<'a>(errors: Vec<ValidationError<'a>>) -> ErrorIterator<'a> {
+    let boxed_errors: ErrorIterator<'a> = Box::new(errors.into_iter());
```

Comment:
Would it work without binding? I.e. immediately return `Box::new(errors.into_iter())`

Reply:
It doesn't, no. The compiler doesn't want to infer that `std::vec::IntoIter<...>` is `dyn Iterator<...>` without it, which is why I made this small helper method in the first place.

Silly compiler. 😠